rules/xcodeproj.bzl

```diff
@@ -670,6 +692,7 @@ Tags for configuration:
         "project_name": attr.string(mandatory = False),
         "bazel_path": attr.string(mandatory = False, default = "bazel"),
         "scheme_existing_envvar_overrides": attr.string_dict(allow_empty = True, default = {}, mandatory = False),
+        "generate_schemes_for_product_types": attr.string_list(mandatory = False, allow_empty = True, default = [], doc = "Generate schemes only for the specified product types if this list is not empty"),
```

Comment:
The valid values are all valid uniform type identifiers for the product type. They are the keys for the dictionary listed here: https://www.rubydoc.info/gems/xcodeproj/Xcodeproj/Constants under the `PRODUCT_TYPE_UTI`.

Valid examples are `application`,`bundle.unit-test`, `framework`, etc.

Reply:
It would be good if you could include this info in the documentation string. You could say:

"Product types must be valid apple product types, e.g. `application`, `bundle.unit-test`, `framework`."